pandas/tests/frame/test_repr_info.py

```diff
@@ -239,8 +239,8 @@ def test_info_duplicate_columns_shows_correct_dtypes(self):
         frame.info(buf=io)
         io.seek(0)
         lines = io.readlines()
-        assert 'a    1 non-null int64\n' == lines[3]
```

Comment:
can you add a an explict test for this, one that specifically checks the formatting (a bit duplicative of this one), but like it separate